fluent/src/main/java/org/codice/imaging/nitf/fluent/impl/NitfParserInputFlowImpl.java

```diff
@@ -45,6 +45,18 @@ public final NitfParserParsingFlow file(final File inputFile) throws FileNotFoun
         return new NitfParserParsingFlowImpl(nitfReader);
     }
 
+    /**
+     *
+     * {@inheritDoc}
+     */
+    @Override
+    public final NitfParserParsingFlow file(final File inputFile, final long skipTimeout)
```

Comment:
Instead of duplicating this, it'd probably be better if the existing implementation (just above) delegated to this new method with the default skip timeout. That should probably have been exposed as a constant in the core module, but I doubt we did.

Reply:
I had certainly considered delegation here, but it changes each of the extant methods from their current two-line bodies to one-line bodies and necessitates creating a private const. All to add extra calls on the stack.

I can make the change if you want it, but it doesn't seem to be gaining much.